Lesson1/Denis Sulima/task2/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>
+#include <list>
+
+
+int main()
+{
+    int X;
```

Comment:
Please, don't leave variables uninitialized. 
You can easily default initialize it with such synax:
int X {};